data/json/npcs/talk_tags.json

```diff
@@ -1455,14 +1787,29 @@
     "text": [
       "We should probably get going.",
       "We'd better get moving.",
-      "Let's head out.",
```

Comment:
Why are you deleting these other snippets in this section?

Reply:
No problem. Generally things shouldn't be moved around since it tends to mess with the git history. Not a big deal for a few lines like this, though.